### PR TITLE
NMS-14434 Enlinkd Clean Persistence Classes

### PR DESCRIPTION
**Pull request by rssntn67**
* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14434

### CODE REVIEW

File: features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/persistence/api/IpNetToMediaDao.java [hunk 1]
```diff
@@ -44,23 +44,23 @@
...
-    public void deleteBySourceNodeId(Integer nodeId);
+    void deleteBySourceNodeId(Integer nodeId);
 
     /**
      * Finds all {@link IpNetToMedia} instances that have a physical address that is equal to the mac address
      * of a {@code BridgeMacLink} of the addressed {@code node}.
-     * @param nodeId
-     * @return
-     */
+     * @param nodeId "The nodeId"
```
Comment (patrick-schweizer) [inline features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/persistence/api/IpNetToMediaDao.java:62]: I woluld remove this line completely. It says exactly the same as in the code itself. I would only add a comment if it adds additional information.

File: features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/model/IpNetToMedia.java [hunk 2]
```diff
@@ -230,7 +230,7 @@ public Integer getSourceIfIndex() {
     /**
      * <p>setSourceIfIndex</p>
      *
-     * @param ifIndex a {@link java.lang.Integer} object.
+     * @param sourceIfIndex a {@link java.lang.Integer} object.
```
[the remark this answers is not in the document]
Comment (RangerRick) [reply features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/model/IpNetToMedia.java:233]: Yeah, I have taking to just deleting the javadoc comments when I edit existing code if all they contain is the autogenerated info.

**Review by RangerRick — DISMISSED** [review]

File: features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/model/BridgeElement.java [hunk 3]
```diff
@@ -67,13 +67,13 @@ public enum BridgeDot1dStpProtocolSpecification {
...
 		DOT1D_STP_PROTOCOL_SPECIFICATION_IEEE8021AQ(5);
-		private int m_type;
+		private final int m_type;
 
 		BridgeDot1dStpProtocolSpecification(int type) {
 			m_type = type;
 		}
 		
-	    protected static final Map<Integer, String> s_typeMap = new HashMap<Integer, String>();
+	    protected static final Map<Integer, String> s_typeMap = new HashMap<>();
```
[the remark this answers is not in the document]
Comment (rssntn67) [reply features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/model/BridgeElement.java:76]: commit a16f139fc8759394dd64355482d11ecee91c1088 (HEAD -> jira/ar/NMS-14434)
Author: Antonio <rssntn67@yahoo.it>
Date:   Mon Jun 27 09:45:52 2022 +0200

    NMS-14434 Enlinkd Clean Persistence Classes

    Resolve conversation patrick-schweizer
    s_typeMap be TYPE_MAP

File: features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/model/IpNetToMedia.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (rssntn67) [reply features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/model/IpNetToMedia.java:233]: commit 2d70f7972fb29aed26694c1e42a388e2f47bc27e (HEAD -> jira/ar/NMS-14434)
Author: Antonio <rssntn67@yahoo.it>
Date:   Mon Jun 27 10:12:18 2022 +0200

    NMS-14434 Enlinkd Clean Persistence Classes

    Resolve conversation with patrick and ben
    Removed default comment

File: features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/persistence/api/IpNetToMediaDao.java (hunk 1 above)
Comment (rssntn67) [reply features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/persistence/api/IpNetToMediaDao.java:62]: commit 309c8f3599c552bb4f571dd78dfb6e9e4922c5c8 (HEAD -> jira/ar/NMS-14434)
Author: Antonio <rssntn67@yahoo.it>
Date:   Mon Jun 27 10:16:16 2022 +0200

    NMS-14434 Enlinkd Clean Persistence Classes

    Resolve conversation patrick-schweizer
    Removed comment

**Pull request merged**